eval/Containerfile

```diff
@@ -0,0 +1,21 @@
+FROM nvcr.io/nvidia/cuda:12.4.1-devel-ubi9
+
+RUN dnf install -y python3.11 python3.11-devel git python3-pip make automake gcc gcc-c++ \
+    && dnf clean all \
+    && rm -rf /var/cache/*dnf*
+
+WORKDIR /instructlab
+
+COPY eval/requirements.txt .
+RUN python3.11 -m ensurepip
+RUN python3.11 -m pip install --no-cache-dir git+https://github.com/instructlab/eval && python3.11 -m pip install -r requirements.txt
+
+RUN export LD_LIBRARY_PATH="$LD_LIBRARY_PATH:/usr/local/lib/python3.11/site-packages/nvidia/cudnn/lib"
+ENV LD_LIBRARY_PATH="$LD_LIBRARY_PATH:/usr/local/lib/python3.11/site-packages/nvidia/cudnn/lib"
+ENV HOME=/instructlab
+
+RUN chmod -R 777 /instructlab
```

Comment:
This doesn't look very safe

Reply:
you have to trust that users can handle this amount of power 🤣